cmd/cli_test.go

```diff
@@ -108,6 +112,10 @@ var tests = []testCase{
 			"4.15.0-1057-aws",
 			"--kernelversion",
 			"59",
+			"--kernelurls",
```

Comment:
sure, i've added as requested. IMO, not sure I see the benefit though as all tests are appended with `--dryrun` so all we are testing really is that we are getting a few lines of output that match some command line options which is covered in the other tests already. I could put any url in the `--kernelurls` option and it would succeed. The only way to properly test that we can build azure now with the supplied `--kernelurls` option (against the related ubuntu-aws target) is to actually do a real build. Under ubuntu-generic it will fail because the package names don't match (e.g. linux-headers*generic) but will succeed for ubuntu-aws because the headers for azure match the pattern linux-headers*.

Reply:
>  IMO, not sure I see the benefit though as all tests are appended with --dryrun so all we are testing really is that we are getting a few lines of output that match some command line options which is covered in the other tests already.

Good point about `--dryrun`. I suppose it doesn't hurt to leave it in at this point.